metadata-load/src/main/java/uk/ac/ebi/ampt2d/metadata/importer/database/SraObjectsImporterThroughDatabase.java

```diff
@@ -76,7 +137,20 @@ public Analysis importAnalysis(String accession) {
 
     @Override
     public ReferenceSequence importReferenceSequence(String accession) {
-        return super.importReferenceSequence(accession);
+        String[] accnRefNameSplit = accession.split("#", 2);
```

Comment:
The split by '#' is not very intuitive (although documented in another method). Could you please add some method documentation to show what the input of this method looks like, and where does it come from?

Reply:
Added the documentation in https://github.com/EBIvariation/metadata-ws/pull/50/commits/f992ccc9192cdef417bcfafd9a2ea1b554e32493. This is @iyangar's implementation for refname/accession handling in EGA import, and I think it's pretty reasonable